lib/game.rb

```diff
@@ -1,56 +1,59 @@
 require_relative 'board'
-require_relative 'console'
 require_relative 'rules'
+require_relative 'human_player'
+require_relative 'dumb_player'
 
 class Game
-  attr_accessor :board, :console, :rules
+  attr_accessor :board, :rules, :player_one, :player_two
 
-  def initialize(board, console, rules)
+  def initialize(board, rules, display, player_creator)
     @board = board
-    @console = console
-    @current_player = 'X'
     @rules = rules
+    @display = display
+    @player_creator = player_creator
+    @current_player = @player_one
```

Comment:
It's strange that `@player_one` is used here, but it isn't actually set until the `player_setup` method gets called in `start`

Reply:
Cyrus mentioned this as well. I removed it from the initialize method and kept it in the play method because you're right, it initializes as nil.